### ISSUE TITLE
password should be hasehed

### ISSUE THREAD

**Issue by tsuncp** [Cleroth/sp_server#7]
current saving plain password in database..

**Comment by Cleroth** [comment]
Agreed. I will probably recode the entire thing from scratch at some point, so that will definitely be in.
